src/platform/Linux/org.bluez.xml

```diff
@@ -0,0 +1,244 @@
+
```

Comment:
move this file under src/platform/Linux/dbus, we will put all gdbus xml files together

Reply:
Done, thanks